.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
```

Comment:
Is this file needed if we're using prettier?

Reply:
We do it in other repo's so thought so

Reply:
@JoviDeCroock fair enough :+1: